crawler/src/main/java/nl/fhict/happynews/crawler/crawler/TwitterCrawler.java

```diff
@@ -1,7 +1,167 @@
 package nl.fhict.happynews.crawler.crawler;
 
+import nl.fhict.happynews.crawler.model.twitterapi.TweetBundle;
+import nl.fhict.happynews.shared.Post;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import twitter4j.*;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+
 /**
- * Created by Tobi on 27-Mar-17.
+ * Created by Sander on 27/03/2017.
  */
-public class TwitterCrawler {
+@Service
+public class TwitterCrawler extends Crawler<TweetBundle> {
+
+    private Twitter twitter;
+    private String hashTags[];
+    private String hashTag;
+    private final static int AMOUNTOFTWEETS = 200;
```

Comment:
Multiple words in a constant or typically written in SNAKE_CASE, so that would be `AMOUNT_OF_TWEETS` here.